src/panels/lovelace/cards/hui-logbook-card.ts

```diff
@@ -57,12 +58,14 @@ export class HuiLogbookCard extends LitElement implements LovelaceCard {
 
   @state() private _logbookEntries?: LogbookEntry[];
 
-  @state() private _persons = {};
-
   @state() private _configEntities?: EntityConfig[];
 
+  @state() private _userIdToName = {};
+
   private _lastLogbookDate?: Date;
 
+  private _fetchUserDone?: Promise<unknown>;
```

Comment:
Please change this name, this is a promise, not a boolean to signal something is done.

Reply:
And also fix the typing, it is not `unknown`, but 
```ts
{
    storage: Person[];
    config: Person[];
}
```

Reply:
Again, this is exactly [what is in logbook panel](https://github.com/home-assistant/frontend/blob/8a33d174d737052724f5cec49ef26c043f4603f9/src/panels/logbook/ha-panel-logbook.ts#L47). I just thought it's a good/okay practice and copied it.

Anyway, I updated all 3 places (logbook card, panel and 'more-info' dialog) from `_fetchUserDone?: Promise<unknown>` to `_fetchUserPromise?: Promise<void>`.